crates/core/src/v1/params.rs

```diff
@@ -92,6 +92,88 @@ pub enum Value {
     Blob(Vec<u8>),
 }
 
+impl Value {
+    /// Returns `true` if the value is [`Null`].
+    ///
+    /// [`Null`]: Value::Null
+    #[must_use]
```

Comment:
Why is this must_use?

Reply:
tbh, idk, it's the code generated by rust-analyzer